README.md

```diff
@@ -1,137 +1,183 @@
-# Ankr APIs JavaScript SDK
-Compact library for interacting with Ankr APIs.
-
-#### SDK supports following MultiChain methods:
-
-- `getLogs` - logs matching the filter.
-- `getBlocks` - blocks within specified range.
-- `getTransactionsByHash` - transaction(s) by hash.
-- `getAccountBalance` - coin and token balances of the wallet.
-- `getNFTsByOwner` - NFT collectibles of the wallet.
-- `getNFTMetadata` - NFT's contract metadata.
-- `getTokenHolders` - list of token holders.
-- `getTokenHoldersCount` - token holders count history by day.
-- `getCurrencies` - currencies of blockchain.
-
-#### MultiChain support
-- ETH: `"eth"`
-- BSC: `"bsc"`
-- Polygon: `"polygon"`
-- Fantom: `"fantom"`
-- Arbitrum: `"arbitrum"`
-- Avalanche: `"avalanche"`
-- Syscoin NEVM: `"syscoin"`
+# ⚓️ ankr.js
 
-### Acquire an API key
-During the launch stage, Ankr Scan offers free access to the API with no request limits. That is, you can access the API without acquiring an API key. Later on Ankr Scan API will become a part of Ankr Protocol [Premium Plan](https://www.ankr.com/protocol/plan/).
+Compact JavaScript library for interacting with Ankr's [Advanced APIs](https://www.ankr.com/advanced-api/).
 
-## Installation
-```shell
-npm i @ankr.com/ankr.js
-```
-or
-```shell
+## Get started in 2 minutes
+
+#### 1. Install the package from npm
+
+```bash
+# with npm
+npm install @ankr.com/ankr.js
+
+# with yarn
 yarn add @ankr.com/ankr.js
 ```
 
-## Import
-node.js require
+#### 2. Initialize the provider
+
 ```javascript
-const { AnkrscanProvider } = require("@ankr.com/ankr.js");
+import AnkrscanProvider from '@ankr.com/ankr.js';
+
+const provider = new AnkrscanProvider('');
```

Comment:
This is the only way to use this package without passing an API key right? If I don't pass in anything it errors out. I need to pass in an empty string.

If yes, I suggest making this experience cleaner by not requiring people to pass in anything if we do not require people to have an API key right now.